x/emissions/module/abci.go

```diff
@@ -15,10 +15,14 @@ func EndBlocker(ctx context.Context, am AppModule) error {
 	sdkCtx.Logger().Debug(
 		fmt.Sprintf("\n ---------------- Emissions EndBlock %d ------------------- \n",
 			blockHeight))
-
+	moduleParams, err := am.keeper.GetParams(sdkCtx)
+	if err != nil {
+		sdkCtx.Logger().Error("Error Getting module params", err)
+		return err
+	}
 	// Remove Stakers that have been wanting to unstake this block. They no longer get paid rewards
-	RemoveStakes(sdkCtx, blockHeight, am.keeper)
-	RemoveDelegateStakes(sdkCtx, blockHeight, am.keeper)
+	RemoveStakes(sdkCtx, blockHeight, am.keeper, moduleParams.HalfMaxProcessStakeRemovalsEndBlock)
+	RemoveDelegateStakes(sdkCtx, blockHeight, am.keeper, moduleParams.HalfMaxProcessStakeRemovalsEndBlock)
```

Comment:
Add error handling

Reply:
intentionally does not throw errors, so as not to affect consensus

also not relevant for this PR, should have brought this up in #469